packages/fxa-settings/src/components/Settings/InputText/index.tsx

```diff
@@ -85,7 +91,8 @@ export const InputText = ({
       ${
         focussed ? 'border-blue-400 shadow-input-blue-focus' : 'border-grey-200'
       }
-      ${disabled ? 'border-grey-100 bg-grey-10' : 'bg-white'} ${className}`}
+      ${disabled ? 'border-grey-100 bg-grey-10' : 'bg-white'}
+      ${errorText ? 'border-red-700 shadow-input-red-focus' : ''} ${className}`}
```

Comment:
Out of scope really but I think this could be cleaned up a lot with the `classnames` helper.